### DISCUSSION TITLE
Can't upload changelogs using Supply action

### DISCUSSION THREAD

**Discussion by MikeMalec** [fastlane/fastlane#21526] [Q&A]
Hi,
I'm trying to upload changelogs using exact folder structure as in documentation but I get error like this:
---Folder Structure---
 fastlane/metadata/android/en-GB/changelogs/default.txt
 fastlane/metadata/android/pl/changelogs/default.txt
---Error---
Preparing to upload for language 'en-GB'...
[!] Cannot find changelog because no version code given - please specify :version_code.

Is it possible to only use files with default name and not defining version_code?

+-------------------------------
|                    Summary for supply
+-------------------------------
| json_key                                                               | key.json                                     
| package_name                                                    | myapp                                       
| track                                                                     | internal                                      
| track_promote_to                                               | alpha                                          
| rollout                                                                   | 1                                                 
| skip_upload_apk                                                 | true                                            
| skip_upload_aab                                                 | true                                            
| skip_upload_metadata                                       | true                                            
| skip_upload_images                                           | true                                            
| skip_upload_screenshots                                   | true                                             
| release_status                                                      | completed                                  
| metadata_path                                                     | ./fastlane/metadata/***             
| skip_upload_changelogs                                     | false                                            
| sync_image_upload                                             |  false                                           
| track_promote_release_status                           | completed                   
| validate_only                                                         | false                       
| check_superseded_tracks                                  | false                       
| timeout                                                                  | 300                         
| deactivate_on_promote                                      | true                        
| changes_not_sent_for_review                           | false                       
| rescue_changes_not_sent_for_review              | true                        
| ack_bundle_installation_warning                       | false                       
+------------------------------------+

**Comment by angelolloqui** [comment]
Same error here... any idea how to fix? I see the `default.txt` should be working, but it seems it is not
